@ecomplus/storefront-components/src/html/ProductVariations.html

```diff
@@ -53,6 +53,7 @@
       <option
         v-for="optionText in options"
         :value="optionText"
+        :class="`variations__select--${optionText}`"
```

Comment:
Então, pior que agora tem outro problema :smile: 
`variations__select` é outro elemento, temos que respeitar o [BEM](http://getbem.com/naming/):

```suggestion
        :class="`variations__select-option--${optionText}`"
```

Ou até deixamos sem classe mesmo (é meio chato estilizar `<option>` de qualquer forma):

```suggestion
```

> Até tinha comentado com você que se não tivesse classe nenhuma no `<option>` podíamos manter sem mesmo.